### DISCUSSION TITLE
Project's direction, values and governance

### DISCUSSION THREAD

**Discussion by pastk** [organicmaps/organicmaps#8559] [Ideas]
I think the way the project is promoted in app stores, on [omaps.org](https://organicmaps.app/), in [GH readme](https://github.com/organicmaps/organicmaps/blob/master/README.md), etc. leads to [misunderstandings](https://github.com/organicmaps/organicmaps/issues/622#issuecomment-2184982888) which unfortunately have become not uncommon.

I think the current OM description doesn't do a good job at describing values or focus of the project clearly.

E.g. literally A HALF of all lines in [Google Play description](https://play.google.com/store/apps/details?id=app.organicmaps&gl=US) talks about how OM is privacy-focused:

<details>
<summary>expand to read those lines</summary>

	‣ Our free app does not track you, does not have ads, and it needs your support.
	• Free, open-source, no ads, no tracking
	• Respects your privacy
	• No unexpected mobile data charges
	Free from trackers and other bad stuff:
	• No ads
	• No tracking
	• No data collection
	• No phoning home
	• No annoying registration
	• No mandatory tutorials
	• No noisy email spam
	• No push notifications
	• No crapware
	• N̶o̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶e̶s̶ Purely organic
	At Organic Maps, we believe that privacy is a fundamental human right:
	• We protect privacy from Big Tech's prying eyes
	• Stay safe no matter wherever you are
	Zero trackers and only minimally required permissions are found according to Exodus Privacy Report.
	Reject surveillance - embrace your freedom.

</details>

This promotes a strong image of an uncompromised privacy champion focused on needs of users who value it.

But at the same time people are [offered to book a hotel via Kayak.com](https://github.com/organicmaps/organicmaps/pull/6523) or to [sync their bookmarks via iCloud or Google Drive](https://github.com/organicmaps/organicmaps/issues/622#issuecomment-2184950981). Its fine for a map app focused on mainstream users, but its no surprise that after all the "We protect privacy from Big Tech's prying eyes" and "Reject surveillance - embrace your freedom" claims the privacy-conscious folks raise an eyebrow, feel upset or misled even.

This discrepancy between a promoted image and reality doesn't serve the project any good.

I believe an open and honest discussion should be held by a broad community to better understand and define actual values of the project and its direction.

(its not just about privacy, there are many other values and topics to discuss like transparency, governance and role of the community in it, adhearance to FOSS values, moderation, etc. - we can tackle them one by one as we go)

**Comment by biodranik** [comment]
> This promotes a strong image of an uncompromised privacy champion focused on needs of users who value it.

> This discrepancy between a promoted image and reality doesn't serve the project any good.

Can you please clarify how _exactly_ the privacy is compromised in OM? And if it's not, then can you please correct your statement to avoid misleading readers, because it sounds as if the privacy is compromised.

**Reply by HansCronau** [reply]
> (...) it sounds as if the privacy is compromised.

I believe that may be exactly the point. @pastk links to an example where [a comment by yourself](https://github.com/organicmaps/organicmaps/issues/622#issuecomment-2184950981) is then [immediately (mis)understood](https://github.com/organicmaps/organicmaps/issues/622#issuecomment-2184982888) to mean that you, a cofounder of Organic Maps, (would) indeed compromise privacy for ease of use.

You understand how privacy happy users (which I imagine could be most of Organics Maps' users/ambassadors) may get a bit nervous when cofounders and active contributors use sentences such as ["Most users don't care"](https://github.com/organicmaps/organicmaps/issues/622#issuecomment-2184950981) and ["99% of iPhone users want to sync files using iCloud out of the box"](https://github.com/organicmaps/organicmaps/issues/622#issuecomment-2186232928).

You can imagine similar nervousness around the experimental KAYAK referral link @pastk mentioned. Sentences such as ["As KAYAK is a traditional website, be aware that they may have tracking on their website like all travel websites."](https://github.com/organicmaps/organicmaps/pull/6523#issue-1984196670) may indeed sound like a compromise on privacy. Given the aforementioned privacy centric branding of Organic Maps, this can understandibly lead to some questions about the values and governance of the product.

**Reply by pastk** [reply]
Rephrasing from https://github.com/organicmaps/organicmaps/issues/622#issuecomment-2187156848 there are many "grades":

(1) An app that doesn't care about privacy, collects user data, share it with ad providers etc.
...
(5) Doesn't collect data itself, but by default integrates with popular non-privacy friendly services.
(6) Integration with non-privacy friendly services is opt-in.
(7) Integration is opt-in and the app warns users about potential privacy implications.
...
(10) Several integration options with open privacy-respecting and self-hosted services only, non-privacy-friendly services are not endorsed or supported, all potential privacy risks are accurately disclosed, etc. etc.

Reading through a very privacy-centric description one likely to have an impression that OM is 9-10, but the latest developments suggest its more like around 5-7.

**Comment by Friptick** [comment]
First off, IMO @biodranik's combative style of communication is _absolutely fine_ because he is the boss here. :smile: (Not even sarcasm BTW.)

On the substance, I agree with @pastk  that the project's mission statement has a fairly ideology-heavy vibe. Personally I would ditch the silly mention of "pesticides" and the meaningless word "safety" but that is my opinion.

Maybe there is a misunderstanding over the term FOSS. It has a technical definition (which @biodranik is focusing on) but also a more political imperative whereby a project should avoid even _encouraging_ the undermining of privacy by others. I.e. by big tech, for example.

Finally, IMO it is not enough for this project to better define its understanding of privacy. It should also explain **why it is better than OsmAnd**. I have made this point here in the past. There is _already_ a privacy-respecting FOSS mapping app for Android and it is _very_ capable. So what is unique about Organic Maps? I know this question p***es off the maintainer and others, but I for one would genuinely like to know the answer. And if the only answer is "it's simpler", is that really such a strong USP? "Yeah it's not as good, but that other app is too complex for you to understand"? Sorry to be provocative but that seems a weak sales pitch, especially as OsmAnd is in fact not that complicated (if you don't make it so). I personally would so much prefer the maintainers of this app to devote their talents to making OsmAnd even more amazing with even better UX, rather than laboriously duplicating a project that already exists. Unless of course I am completely wrong and there really is something fundamentally different and _better_ about Organic Maps. If so, what it is?

**Reply by pastk** [reply]
re comparison to Osmand while its a good question its maybe not exactly on-topic here and there were discussions about it already (e.g. https://github.com/orgs/organicmaps/discussions/6029 and probably there are more)

**Comment by Misalf-git** [comment]
> > This promotes a strong image of an uncompromised privacy champion focused on needs of users who value it.
> 
> > This discrepancy between a promoted image and reality doesn't serve the project any good.
> 
> Can you please clarify how _exactly_ the privacy is compromised in OM? And if it's not, then can you please correct your statement to avoid misleading readers, because it sounds as if the privacy is compromised.

Concern about a potential bad appearance doesn't sound like a claim to me.
"Correcting" this expression does not compute.

**Comment by pastk** [comment]
@biodranik for example I know that orientation on a typical (non-advanced and prob not caring about privacy much) user is very important to you.
Then how come it wasn't made clear in the description?
The only line in the aforementioned GPlay description that seem relevant is
> • Simple to use, with only most important features included

(compared to 20 lines focused on privacy)

**Comment by pastk** [comment]
This overemphasis on privacy in store descriptions seems to come from the very inception of OM when it was forked from Maps.Me. Back then it was basically the main thing that differed OM from MM, hence it was emphasized so much.

Nowadays OM is an established independent project, so we can author a more balanced description (which if fact reflects project's direction and priorities) which will properly highlight various advantages of OM (like simplicity and easy of use...) and a balance between them.

So I urge everyone to focus on what actually unites us (and respect and value our differences at the same time) and share how do you see OM, its direction and priorities 🙏

Then based on this we can come up with improved and clear mission statements, priority lists, descriptions, etc..

**Comment by rtsisyk** [comment]
After yesterday's incident, it is clear that the Organic Maps project was seized by Mr. Alexander Borsuk, a co-founder of MAPS.ME. He covertly changed the license, proprietarized the open-source code, and even began booting people from the project to suppress dissenting voices.

**Reply by HansCronau** [reply]
Could you provide some context?

Also, just to check: You are aware that Alexander Borsuk (@biodranik), who indeed co-founded Maps.Me, was also one of the people who later forked Maps.Me to create Organic Maps?

**Reply by langfingaz** [reply]
See https://github.com/orgs/organicmaps/discussions/9837

**Comment by pastk** [comment]
Let's define Core Values of the project together:
- https://github.com/organicmaps/organicmaps/pull/9905
